### ISSUE TITLE
[BUG] retopoflow 3 causes blender to open infinite number of instances 

### ISSUE THREAD

**Issue by gotenxds** [CGCookie/retopoflow#855]
Please prefix your issue name with one of the following: **[BUG]** **[PROPOSAL]** **[QUESTION]**

Retopoflow Version: latest from b280 branch

Blender Version/Hash: latest from steam (2.8.3)

Platform Version/Distribution: windows 10 64bit

**Issue:**

Adding retopoflow 3 to blender works fine until you close blender once and reopen it, then for some reason blender starts up again and again and again forever until the pc crashes 

{*add any screenshots of the issue here*}

Kinda hard to take screenshots when everything crashes, sorry.

I did manage to duplicate this in a non steam install and on 2 different pc's (both are windows)

&nbsp;

**How to Reproduce:**
- Add retofolow 3 to blender
- close blender
- open blender
- try and close blender asap or your pc will crash in less than 30seconds

{*attach any files needed to replicate the issue*}

**Comment by vxlcoder** [comment]
Thanks for the report, although it is very strange. Do you have any other add-ons enabled? Can you try a "load factory settings" in blender, save, then follow your steps again? Has RF worked correctly before but not now? I have tested RF extensively on multiple platforms and blender versions and I've never seen this happen.

**Comment by gotenxds** [comment]
@vxlcoder Hi vxlcoder, thx for the quick response, I tried what you suggested, did not solve the issue
Made a small video of what happens (sorry for the weird resolution I dont have a good setup for recording my screen atm)

https://drive.google.com/file/d/1Y26OsC4g3S2cyZjprZjynv9xDabCgGdM/view?usp=sharing

This is a clean install of blender, no addons, as I stated, it worked for the first install, even used it for a bit, then I closed blender and opened it up and this happens, now it won't work at all, after installing the addon the issue happens again.

**Comment by vxlcoder** [comment]
Thanks for response! Will take a look as soon as I can!

**Comment by vxlcoder** [comment]
I think I might have a cause figured out. I'll make a change tomorrow that, if you don't mind, I'd like you to try to see if it solves the issue. Thanks for the screen capture! (BTW, OBS studio is great screen recording tool if you want to capture anything in the future)

**Comment by gotenxds** [comment]
Sure we'd love to test it out. waiting on an update

**Comment by vxlcoder** [comment]
sorry for my late response, @gotenxds!

are you comfortable making a _very_ small change to the code?  I can tell you exactly what to change to see if it improves your experience.  otherwise, i can make and push the change, but you making the change might go faster for you.

**Comment by vxlcoder** [comment]
if you're comfortable, try this... open up `[addons_folder]/[retopoflow_addon]/retopoflow/retopoflow.py` in a text editor, and change line 63 from `version='process'` to `version='thread'`.

**Comment by gotenxds** [comment]
@vxlcoder 
Hi, made the change, works great!
Thanks a lot!

**Comment by vxlcoder** [comment]
thanks for the report, and glad it's working now!

very strange that the previous version was causing problems... i wonder if it has anything to do with downloading blender from steam.  if you wouldn't mind, would you try downloading blender 2.83 from blender.org instead, either re-install RF or revert the change you just made, and see if you get the same issue.  this might be something i need to report to blender core devs

**Comment by gotenxds** [comment]
Sure, we checked and if we revert the change the same happens in a non-steam version just the same, latest blender version from the blender site as of 10m ago.

**Comment by vxlcoder** [comment]
interesting.  thanks!  let us know if you have any other problems, but i'll close this issue for now.  cheers!
